### PR TITLE
testPullRequestReviews

### PR DESCRIPTION
**Pull request by gsmet**
## test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Java API for GitHub
```
Comment (gsmet) [inline README.md:1]: Some niggle

**Review by gsmet — COMMENTED** [review]
Some review comment

File: README.md (hunk 1 above)
Comment (maximevw) [inline README.md:1]: Some niggle

File: README.md [hunk 2]
```diff
@@ -2,2 +1,4 @@
+# Java API for GitHub TEST (stable)
```
Comment (maximevw) [inline README.md:2]: A single line comment

File: README.md [hunk 3]
```diff
@@ -2,2 +1,4 @@
...
 
 See https://github-api.kohsuke.org/ for more details
```
Comment (maximevw) [inline README.md:3]: A multiline comment

**Review by maximevw — COMMENTED** [review]
Some review comment

File: README.md (hunk 1 above)
Comment (maximevw) [inline README.md:1]: Some niggle

File: README.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
-Java API for GitHub
+# Java API for GitHub TEST (stable)
```
Comment (maximevw) [inline README.md:2]: A single line comment

File: README.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
...
 
 See https://github-api.kohsuke.org/ for more details
```
Comment (maximevw) [inline README.md:3]: A multiline comment

**Review by maximevw — COMMENTED** [review]
Some review comment

**Pull request closed without merging**